docs/_includes/components/footer.html

```diff
@@ -45,8 +45,16 @@ <h5 class="footer-title m-b-2">Learn more</h5>
             <h5 class="footer-title m-b-2">Follow us</h5>
             <ul class="list-unstyled">
               <li>
-                <a href="#" class="link-icon" target="_blank"><i class="icon icon-lg icon-facebook-inverse"></i></a>
-                <a href="#" class="link-icon" target="_blank"><i class="icon icon-lg icon-twitter"></i></a>
+                <a href="#" class="link-icon" target="_blank" rel="noopener noreferrer"><i class="icon icon-lg icon-facebook-inverse"></i></a>
+                <a href="#" class="link-icon" target="_blank" rel="noopener noreferrer"><i class="icon icon-lg icon-twitter"></i></a>
+                <a href="https://www.instagram.com/transferwise" class="link-icon" target="_blank" rel="noopener noreferrer">
```

Comment:
why do we add instagram url and not facebook nor twitter?

Reply:
Agree. I remove the instagram url